wgpu-core/src/command/render.rs

```diff
@@ -605,6 +614,41 @@ impl<'a, A: HalApi> RenderPassInfo<'a, A> {
         let mut extent = None;
         let mut sample_count = 0;
 
+        let mut detected_mutiview: Option<Option<NonZeroU32>> = None;
+
+        let mut check_multiview = |view: &TextureView<A>| {
+            // Get the multiview configuration for this texture view
+            let layers = view.selector.layers.end - view.selector.layers.start;
+            let this_multiview = if layers == 1 {
+                None
+            } else {
+                let layers = NonZeroU32::new(layers).expect("Non-zero layers expected");
```

Comment:
this doesn't need to check for view layers being zero. View creation already does.
So this code could be just `NonZeroU32::new(layers)`

Reply:
Did you mean `NonZeroU32::new_unchecked(layers)`?